code/style.css

```diff
@@ -1,7 +1,339 @@
+}
+
+.submit:active {
+  background-color: #3e8e41;
+  box-shadow: 8px 8px 10px #666;
+  transform: translateY(4px);
+}
+
+/* gradient-color on the button changes when hover */
+
+.submit:hover{
+  background: linear-gradient(-45deg, #a8a377, white, #a8a377);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+}
+
+@keyframes gradient {
+  0% {
+      background-position: 0% 50%;
+  }
+  50% {
+      background-position: 100% 50%;
+  }
+  100% {
+      background-position: 0% 50%;
+  }
+}
+
+/*The Accordion Section*/
+
+.accordion-collumn {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: lavender;
+  margin: 10px;
+  border-radius: 5px;
+  height: 900px;
+  
+}
+
+.accordion-header {
+  font-family: 'Syncopate', sans-serif;
+  margin: 25px auto;
+  font-size: 30px;
+}
+
+.accordion {
+  width: 90%;
+  max-width: 1000px;
+  margin: 2rem auto;
+}
+
+.accordion-item {
+  background-color: white;
+  color: black;
+  margin: 15px 0;
+  border-radius: 5px;
+  box-shadow: 0 2px 5px 0 rgba(56, 43, 82,0.5)
+}
+
+.accordion-item-header {
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+  padding: 8px 48px 8px 16px;
+  min-height: 50px;
+  line-height: 1.25rem;
+  font-weight: bold;
+}
+
+.accordion-item-header::after {
+  content: "\002B";
+  font-size: 30px;
+  position: absolute;
+  right: 16px;
+}
+
+.accordion-item-header.active::after {
+  content: "\2212";
+}
+
+.accordion-item-body {
+  padding: 20px;
+  line-height: 25px;
+  border-top: 1px solid;
+  border-image: linear-gradient(to right, transparent, #747982, transparent)1;
+}
+
+/*Makes the FAQ questions open and close*/
+
+.answer {
+  display: none;
+}
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+
+footer {
+  height: 80px;
+  font-size: 25px;
+  font-family: 'Syncopate', sans-serif;
+  text-align: center;
+  color:#a8a377;
+  padding-top: 50px;
+  background-color: teal;
 }
+
+/*Tablet View*/
+
+@media(min-width: 768px) {
+
+.slogan {
+    font-size: 50px;
+    width: 250px;
+  }
+
+  .logo {
+    width: 250px;
+    margin-top: 30vh;
+  }
+
+.form-and-faq-container {
+  flex-direction: row;
+}
+
+.form {
+  width: 50%;
+  margin: 10px 5px 10px 10px;
+  height: auto;
+}
+
+input {
+  font-size: 25px;
+}
+
+.accordion-collumn {
+  width: 50%;
+  margin: 10px 10px 10px 5px;
+  height: auto;
+}
+
+.accordion-header {
+  margin: 100px auto 80px auto;
+}
+
+.form-header {
+  margin: 100px auto 80px auto;
+}
+}
+
+/*Desktop View*/
+
+@media(min-width: 992px) {
+  .slogan {
+    font-size: 80px;
+    width: 250px;
+  }
+
+  .contact-booking {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .radio-condition {
+    margin-right: 110px;
+  }
+
+  input {
+    font-size: 20px;
+  }
+
+  .submit {
+    margin-right: 400px;;
+  }
+
+  .form {
+    padding-left: 70px;
+  }
+}
+
+
+/* font-family: 'Courgette', cursive;
+font-family: 'Ma Shan Zheng', cursive; */
```

Comment:
If you do not need this code, try to delete it before you submit the final version :)